bin/protoc-diff

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
+
+set -eu
+
+# Install unzip if not already installed (Linux only)
+system=$(uname -s)
+if [ "$system" = Linux ] && [ $(dpkg-query -W -f='${Status}' unzip 2>/dev/null | grep -c "ok installed") -eq 0 ];
+then
+  apt-get update
+  apt-get install unzip
+fi
+
+bin/protoc-go.sh
+
+dir_dirty=$(git status -s)
```

Comment:
This is going to pick up untracked files. Funny enough, I had untracked files when testing and this was failing and I didn't see why right away.

What we  probably should check for here is just if any files have changed with `git diff HEAD`.

Reply:
if the protoc generates new bindings files which aren't already checked into git, they would be untracked files, right?  Isn't that exactly what we want to catch?  Since this is intended to be run in CI, I don't think there would be any other untracked files hanging around.

Reply:
Yea that's true. This was also coming from the fact that we use `diff-index` (which I think `diff` is just recommend now) in `_tag.sh`. Similar situation so thought I'd suggest it. Definitely not a blocker.